kratos/tests/materials_with_subproperties.json

```diff
@@ -0,0 +1,111 @@
+                    "DENSITY"       : 7850.0,
+                    "YOUNG_MODULUS" : 206900000000.0,
+                    "POISSON_RATIO" : 0.29
+                },
+                "Tables"           : {}
+            },"sub_properties" : [{
+                "properties_id"   : 21,
+                "Material"        : {
+                    "constitutive_law" : {
+                        "name" : "HyperElastic3DLaw"
+                    },
+                    "Variables"        : {
+                        "THICKNESS"     : 0.000889,
+                        "DENSITY"       : 7850.0,
+                        "YOUNG_MODULUS" : 206900000000.0,
+                        "POISSON_RATIO" : 0.29
+                    },
+                    "Tables"           : {}
+                }
+            },{
+                "properties_id"   : 22,
+                "Material"        : {
+                    "constitutive_law" : {
+                        "name" : "LinearElastic3DLaw"
+                    },
+                    "Variables"        : {
+                        "THICKNESS"     : 0.000889,
+                        "DENSITY"       : 7850.0,
+                        "YOUNG_MODULUS" : 206900000000000.0,
+                        "POISSON_RATIO" : 0.0
+                    },
+                    "Tables"           : {}
+                }
+            },{
+                "properties_id"   : 2
+            }]
+        },{
+            "properties_id"   : 12,
+            "Material"        : {
+                "constitutive_law" : {
+                    "name" : "LinearElastic3DLaw"
+                },
+                "Variables"        : {
+                    "THICKNESS"     : 0.000889,
+                    "DENSITY"       : 7850.0,
+                    "YOUNG_MODULUS" : 206900000000000.0,
+                    "POISSON_RATIO" : 0.0
+                },
+                "Tables"           : {}
+            }
+        },{
+            "properties_id"   : 13,
+            "Material"        : {
+                "constitutive_law" : {
+                    "name" : "HyperElastic3DLaw"
+                },
+                "Variables"        : {
+                    "THICKNESS"     : 0.000889,
+                    "DENSITY"       : 7850.0,
+                    "YOUNG_MODULUS" : 206900000000000.0,
+                    "POISSON_RATIO" : 0.0
+                },
+                "Tables"           : {}
+            }
+        }]
+    },{
+        "model_part_name" : "Outlet",
+        "properties_id"   : 2,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "LinearElastic3DLaw"
+            },
+            "Variables"        : {
+                "THICKNESS"     : 0.000889,
+                "DENSITY"       : 7850.0,
+                "YOUNG_MODULUS" : 206900000000000.0,
+                "POISSON_RATIO" : 0.0
+            },
+            "Tables"           : {}
+        }
+    }]
+}
+
```

Comment:
the proposal of @KratosMultiphysics/technical-committee  and specifically of @maceligueta was to give the possibility to assign one pointer to an existing property as a subproperty

Reply:
This is done in fact in the test, check it please

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/80332ee8dec355c6d78e9d153df521c68f90d000/kratos/tests/materials_with_subproperties.json#L63

You can create a new subproperty or assign existing ones

Reply:
No, I prefer to have a common interface, I think is more straightforward, that's why I put the checks, to avoid duplicated properties

Reply:
Ok let's speak about this.

@KratosMultiphysics/technical-committee opinions?

others?